workers/unity/Packages/io.improbable.gdk.debug/WorkerInspector/Codegen/ComponentVisualElement.cs

```diff
@@ -20,6 +21,9 @@ protected ComponentVisualElement()
 
             ComponentFoldout = this.Q<Foldout>(className: "component-foldout");
             AuthoritativeToggle = this.Q<Toggle>(className: "is-auth-toggle");
+
+            ComponentFoldout.text = "Component";
```

Comment:
Had to add this to generate the text element. If I don't the text element is generated after the button which aligns it to the right.

Reply:
Can we add an inline comment stating this?